core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/provider.go

```diff
@@ -318,7 +318,7 @@ func (p *logEventProvider) updateFiltersLastPoll(entries []upkeepFilter) {
 	p.filterStore.UpdateFilters(func(orig, f upkeepFilter) upkeepFilter {
 		if f.lastPollBlock > orig.lastPollBlock {
 			orig.lastPollBlock = f.lastPollBlock
-			p.lggr.Debugw("Updated lastPollBlock", "lastPollBlock", f.lastPollBlock, "upkeepID", f.upkeepID)
+			// p.lggr.Debugw("Updated lastPollBlock", "lastPollBlock", f.lastPollBlock, "upkeepID", f.upkeepID)
```

Comment:
this log is pretty useful to debug progress (and helped catch a bug). Maybe we can log this on some sample (like 1%)?